prow/jobs/incubator/reconciler/reconciler.yaml

```diff
@@ -275,6 +275,79 @@ periodics: # runs on schedule
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: periodic-main-kyma-incubator-reconciler-auto-imagebump
+      annotations:
+        description: "Reconciler image bump in control-plane"
+        pipeline.trigger: "periodic"
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "periodic-main-kyma-incubator-reconciler-auto-imagebump"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-log-collector-slack-token: "true"
+      cron: "0 1-22/4 * * 1-5"
+      skip_report: false
+      decorate: true
+      decoration_config:
+        grace_period: 600000000000
+        timeout: 14400000000000
+      cluster: untrusted-workload
+      extra_refs:
+        - org: kyma-project
+          repo: control-plane
+          path_alias: github.com/kyma-project/control-plane
+          base_ref: main
+        - org: kubernetes
+          repo: test-infra
+          path_alias: github.com/kubernetes/test-infra
+          base_ref: master
+          base_sha: 3fba7110c39a7a0e7fb087dbb4af0674cafd341c
```

Comment:
Why do we need this sha?

Reply:
As I am using the master branch from k8s, so IMO its better to bind to a specific commit so that later changes wont effect our job? What do you think?

Reply:
That sounds good 👍